static/app/views/organizationStats/teamInsights/teamStability.tsx

```diff
@@ -180,35 +193,88 @@ class TeamStability extends AsyncComponent<Props, State> {
     const groupedProjects = groupByTrend(sortedProjects);
 
     return (
-      <StyledPanelTable
-        isEmpty={projects.length === 0}
-        headers={[
-          t('Project'),
-          <RightAligned key="last">{tct('Last [period]', {period})}</RightAligned>,
-          <RightAligned key="curr">{t('Last 7 Days')}</RightAligned>,
-          <RightAligned key="diff">{t('Difference')}</RightAligned>,
-        ]}
+      <SessionsRequest
+        api={api}
```

Comment:
can we reuse `this.api` that AsyncComponent i think has instead of making a new one

Reply:
👍